java/src/jmri/jmrit/display/layoutEditor/configurexml/LayoutSlipXml.java

```diff
@@ -38,6 +38,14 @@ public Element store(Object o) {
         element.setAttribute("ident", p.getName());
 
         element.setAttribute("slipType", "" + p.getSlipType());
+
```

Comment:
This is why I appreciate the code review… I was under the (miss-)impression that not saving the turnout & secondturnout attributes was a bug… not understanding that these are saved as 'turnout' and 'turnoutB' elements for slips. This will be reverted ASAP (next commit) also.